src/Types.hs

```diff
@@ -265,8 +265,11 @@ adjustIngredients factor =
   List.map (\ingr -> ingr { quantity = quantity ingr * factor })
 
 showRecipe :: (String -> String) -> Recipe -> Maybe Int -> RichText
-showRecipe t r maybeServings =  showRecipeHeader t r maybeServings
-                ~~ "\n" ~~ List.unlines (showRecipeSteps r)
+showRecipe t r maybeServings =
+  if maybeServings == Nothing
```

Comment:
The more idiomatic Haskell would be:
```haskell
  case maybeServings of
    Nothing -> "Error"
    Just servings -> undefined -- do something with "servings" here
```
The reason why we tend to do it this way is that in the "Just" branch, you know that there is a value in there, and the `case` lets you go ahead and get it out.

Reply:
Thanks for the guidance!

`showRecipeHeader` expects a `Maybe Int` and a `case` statement would return an `Int`. I can wrap the `servings` with a `Just` (seems a little dirty?) or I can refactor `showRecipeHeader` to take in an `Int` instead. This would require a change to the following line:
https://github.com/JackMiranda/herms/blob/36d78bc4eaa4aac4d3cad5fcaff40fb9ea634d10/app/herms.hs#L223

Which one is the better option?

Reply:
I went ahead with the lazy option of wrapping `servings` and in `Just` and passing it to `showRecipesSteps`. I will revert it if a rework of `showRecipeHeader` is preferred

Reply:
@ppartarr Ah, good point. This seems like a good solution for now.

It's a bit of an edge case... I rarely work with code that has to figure out if something is a `Just` or a `Nothing` but doesn't need to use what's in the `Just`.